cmake/toolchain/cross-compile/Kconfig

```
@@ -0,0 +1,10 @@
+# Copyright © 2022 Keith Packard
+# SPDX-License-Identifier: Apache-2.0
+
+config TOOLCHAIN_CROSS_COMPILE_SUPPORTS_THREAD_LOCAL_STORAGE
+	prompt "Cross-compile toolchain supports TLS"
+	def_bool n
```

default value for a bool is `n`, so `bool` is enough, which again allows you to skip the `prompt` part.

Can be simplified to:
```suggestion
	bool "Cross-compile toolchain supports TLS"
```

Ref: https://docs.zephyrproject.org/latest/build/kconfig/tips.html#redundant-defaults

---

thanks, that's a bit cleaner looking.